packages/hebao_v1/contracts/modules/core/ForwarderModule.sol

```diff
@@ -48,11 +50,11 @@ contract ForwarderModule is BaseModule
     }
 
     bytes32 constant public META_TX_TYPEHASH = keccak256(
-        "MetaTx(address from,address to,uint256 nonce,address gasToken,uint256 gasPrice,uint256 gasLimit,bytes32 txAwareHash,bytes data)"
+        "MetaTx(address from,address to,uint256 nonce,address gasToken,uint256 gasPrice,uint256 gasLimit,bytes32 txAwareHash,bytes32 dataHash)"
```

Comment:
Why this change? When signing with EIP712 `data` won't be visible any more. `bytes` should work fine.

Reply:
because the data was converted to bytes32 by keccak256(data_), If not change do this change, the signature will fail.

Reply:
Or I can change the  keccak256(data) back to data.

Reply:
Hmmm... `keccak256(data)` is correct for calculating the hash because the `bytes` field is hashed internally by EIP712 to a bytes32 value. Not sure why it wouldn't work. Will try it.

Reply:
Pushed a commit that changes it to `bytes`, seemingly successful. Please revert if you come across any issues with it for some reason.